persist-cli-tests/src/test/resources/test-src/input/tool_test_generate_65/persist/model.bal

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/persist as _;
+
+type Book record {|
+    readonly string bookId;
+    string title;
+    string author;
+    decimal price;
+    int stock;
+	OrderItem? orderitem;
+|};
+
+type Order record {|
+    readonly string orderId;
+    string customerId;
+    string createdAt;
+    decimal totalPrice;
+	OrderItem[] orderItems;
+	Payment? payment;
```

Comment:
Shall we format this file.

Reply:
Fixed in [dbb9bdf](https://github.com/ballerina-platform/persist-tools/pull/313/commits/dbb9bdfde426cbb5298aff9b6d0c520990fefb9b)